clients/rospy/src/rospy/rosconsole.py

```diff
@@ -33,15 +33,23 @@
 from __future__ import print_function
 
 import os
+import re
 import socket
 import sys
 
+from datetime import datetime
+from dateutil.tz import tzlocal
+
+from argparse import ArgumentParser
```

Comment:
Sorry, for not being clearer. I wasn't referring to the location of the import statement. Instead I was trying to point out that the first command line argument of `rosconsole` is inspected "manually" by looking at `sys.argv` and then `argparse` is only used for the new `echo` verb. It would be nice if `argparse` would be used for all arguments including the decision/selection based on the verb.

But that is clearly outside the scope of this PR.

Reply:
Gotcha! Yes, that makes a lot of sense, but definitely not a change for this PR.